server.js

```diff
@@ -3,9 +3,43 @@ require('habitat').load();
 var express = require('express');
 var app = express();
 var routes = require('./routes');
+var compression = require('compression');
+var helmet = require('helmet');
+var csp = require('helmet-csp');
+var frameguard = helmet.frameguard;
 
 var bodyParser = require('body-parser');
+
+app.set('trust proxy', true);
+
 app.use(bodyParser.json());
+app.use(compression());
+app.use(helmet());
```

Comment:
lets be more verbose about the security settings we use.

frameguard: https://helmetjs.github.io/docs/frameguard/ - lets set it to 'deny'
hidePoweredBy: https://helmetjs.github.io/docs/hide-powered-by/ just use the default
hsts: https://helmetjs.github.io/docs/hsts/ - max age of one year
ieNoOpen: https://helmetjs.github.io/docs/ienoopen/ - use default
noSniff: https://helmetjs.github.io/docs/dont-sniff-mimetype/ - use default
xssFilter: https://helmetjs.github.io/docs/xss-filter/ - use default